src/components/InputOption.js

```diff
@@ -4,8 +4,18 @@ import {
   Form,
   Row
 } from 'react-bootstrap'
+import debounce from 'lodash/debounce'
 
 class InputOption extends Component {
+  constructor(props) {
+    super(props);
+    this.delayedSet = debounce( (value, key, onChange) => {
+      let attrs = {}
+      attrs[key] = value
+      onChange(attrs)
+    }, 1500)
```

Comment:
WDYT about this timeout? It is a bit long, but I liked it not computing while I was typing.

Reply:
👍🏻